src/components/pages/UsersAndTransects/UsersAndTransects.js

```diff
@@ -441,6 +434,46 @@ const UsersAndTransects = () => {
     handleSetTableUserPrefs({ propertyKey: 'methodsFilter', currentValue: methodsFilter })
   }, [methodsFilter, handleSetTableUserPrefs])
 
+  const calcUserCollectRecordCount = (tableData, user) => {
+    const userTableCellData = tableData.map((cellData) => cellData[user])
+
+    return userTableCellData.reduce((accumulator, userCollectRecord) => {
+      if (userCollectRecord !== '-') {
+        const collectRecordCount =
+          userCollectRecord !== '-'
```

Comment:
recommend avoiding redundant checks - `userCollectRecord !== '-'` is added twice